pkg/diagnostics/grpc_tracing.go

```diff
@@ -100,7 +101,8 @@ func GRPCTraceUnaryServerInterceptor(appID string, spec config.TracingSpec) grpc
 // GRPCTraceStreamServerInterceptor sets the trace context or starts the trace client span based on request.
 func GRPCTraceStreamServerInterceptor(appID string, spec config.TracingSpec) grpc.StreamServerInterceptor {
 	return func(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
-		if strings.Index(info.FullMethod, daprPackagePrefix) == 0 {
+		fmt.Printf("*** GOT STREAM REQUEST FOR %s\n", info.FullMethod)
+		if strings.Index(info.FullMethod, daprPackagePrefix) == 0 || strings.Index(info.FullMethod, durableTaskPackagePrefix) == 0 {
```

Comment:
This change was required to get past the first issue (without this, we get an error complaining about a missing `dapr-app-id` metadata header).